lib/karafka.rb

```diff
@@ -32,6 +35,11 @@ def monitor
       @monitor ||= App.config.monitor
     end
 
+    # @return [::Karafka::EventPublisher] event publisher we want to use.
+    def events
+      @events ||= EventPublisher.new
```

Comment:
Could you add following note to the yard doc:

We don't need to ensure that it is not executed in a thread for the first time (assingment is not thread-safe) as it gets used during the non-threaded boot process.

Reply:
I guess we could also use their aproach from the docs: http://dry-rb.org/gems/dry-events/ - in terms of the API usage and event registration. Especially as I requested to raise an error on non registered events.

Reply:
Could you be more specific? Are you suggesting removing the `EventPublisher` class and including the Publisher mixin in `Karafka::App` instead?

Reply:
Giving it a second though. Please ignore.